smart-contracts/contracts/DigitalaxPodePortal.sol

```diff
@@ -0,0 +1,185 @@
+    event DigitalaxPodePortalContractDeployed();
+    event DigitalaxPodePortalMetadataAdded(uint256 index, string tokenUri);
+    event DigitalaxPodePortalMinted(uint256 tokenId, address beneficiary, string tokenUri);
+
+    /// @dev Required to govern who can call certain functions
+    DigitalaxAccessControls public accessControls;
+    DigitalaxPodeNFT public podeNft;
+
+    /// @dev current max tokenId
+    uint256 public tokenIdPointer;
+
+    /// @dev Limit of tokens per address
+    uint256 public maxLimit = 1;
+
+    /// @dev TokenID -> Designer address
+    mapping(uint256 => address) public designers;
+
+    /// @dev List of metadata
+    string[] public metadataList;
+
+    /// @dev ERC721 Token ID -> ERC1155 ID -> Balance
+    mapping(uint256 => mapping(uint256 => uint256)) private balances;
+
+    /**
+     @param _accessControls Address of the Digitalax access control contract
+     */
+    constructor(DigitalaxAccessControls _accessControls, DigitalaxPodeNFT _podeNft) public {
+        accessControls = _accessControls;
+        podeNft = _podeNft;
+        emit DigitalaxPodePortalContractDeployed();
+    }
+
+    /**
+     @notice Mints a DigitalaxPodePortal AND when minting to a contract checks if the beneficiary is a 721 compatible
+     @dev Only senders with either the minter or smart contract role can invoke this method
+     @param _beneficiary Recipient of the NFT
+     @return uint256 The token ID of the token that was minted
+     */
+    function mint(address _beneficiary) external returns (uint256) {
+        require(balanceOf(_msgSender()) < maxLimit, "DigitalaxPodePortal.mint: Sender already minted");
+        require(podeNft.balanceOf(_msgSender()) > 0, "DigitalaxPodePortal.mint: Sender must have PODE NFT");
+
+        // Valid args
+        uint256 _randomIndex = _rand();
+        require(bytes(metadataList[_randomIndex]).length > 0, "DigitalaxPodePortal.mint: Token URI is empty");
+
+        tokenIdPointer = tokenIdPointer.add(1);
+        uint256 tokenId = tokenIdPointer;
+
+        // Mint token and set token URI
+        _safeMint(_beneficiary, tokenId);
+        _setTokenURI(tokenId, metadataList[_randomIndex]);
+
+        emit DigitalaxPodePortalMinted(tokenId, _beneficiary, metadataList[_randomIndex]);
+        return tokenId;
+    }
+
+    /**
+     @notice Burns a DigitalaxPodePortal, releasing any composed 1155 tokens held by the token itself
+     @dev Only the owner or an approved sender can call this method
+     @param _tokenId the token ID to burn
+     */
+    function burn(uint256 _tokenId) external {
+        address operator = _msgSender();
+        require(
+            ownerOf(_tokenId) == operator || isApproved(_tokenId, operator),
+            "DigitalaxPodePortal.burn: Only garment owner or approved"
+        );
+
+        // Destroy token mappings
+        _burn(_tokenId);
+
+        // Clean up designer mapping
+        delete designers[_tokenId];
```

Comment:
This means you need to set designers[_tokenId] somewhere. That looks to be missing in "mint"